plugins/rapid7_intsights/unit_test/test_get_cve_by_id.py

```diff
@@ -18,31 +19,44 @@ def setUpClass(cls, mock_request) -> None:
 
     @patch("requests.request", side_effect=Util.mock_request)
     def test_get_cve_by_id_empty(self, make_request):
+        validate({}, GetCveByIdInput.schema)
         actual = self.action.run()
         expected = Util.read_file_to_dict("expecteds/get_cve_by_id_all.json.resp")
         self.assertEqual(expected, actual)
+        validate(actual, GetCveByIdOutput.schema)
 
     @patch("requests.request", side_effect=Util.mock_request)
     def test_get_cve_with_offset(self, make_request):
-        self.maxDiff = None
-        actual = self.action.run({Input.CVE_ID: ["CVE-2021-7064"]})
+        input_params = {Input.CVE_ID: ["CVE-2021-7064"]}
+        validate({}, GetCveByIdInput.schema)
```

Comment:
copy and paste issue? validating `{}` and not input_params? On this test and the next couple

Reply:
yeah good catch, this copy paste error has been addressed